docs/concierge-commandline.md

```diff
@@ -85,8 +72,11 @@ time, you can set the `org.osgi.framework.storage.clean` property to `onFirstIni
 The `org.eclipse.concierge.profile` allows you to create a separate storage directory
 for each profile.
 
-The `repo` property is declared pointing to the Apache Felix repository, and is used later in the commands as `${repo}`.
+The `repo` property is declared pointing to the Apache Felix repository, and is used later
+in the commands as `${repo}`.
+
+Finally all bundles are installed and started with the `-istart` command. This can take 
+both a web URL or a file system URL.
+
 
-Finally all bundles are installed and started with the `-istart` command or using `-install` and `-start` command. This can take both a web URL or a file system URL.
 
-If you specify a `-initlevel <level>` all bundles installed or started later will start with the given start level.
```

Comment:
I think the docs you used have been outdated docs from Website. We should use the version from the old "docs" folder